.circleci/config.yml

```diff
@@ -507,6 +507,116 @@ jobs:
+    working_directory: ~/marsha/src/aws/lambda-encode
+    steps:
+      - checkout:
+          path: ~/marsha
+      - restore_cache:
+          keys:
+            - v3-lambdas-encode-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install encode lambda dependencies
+          command: yarn install --frozen-lockfile
+      - run:
+          name: lint code
+          command: yarn prettier --write --list-different '**/*.+(json|js)'
+
+  lint-lambda-complete:
+    docker:
+      - image: circleci/node:12.20
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+      # Run on the highest version of node available on AWS Lambda
+      # https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime
+    working_directory: ~/marsha/src/aws/lambda-complete
+    steps:
+      - checkout:
+          path: ~/marsha
+      - restore_cache:
+          keys:
+            - v3-lambdas-complete-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install complete lambda dependencies
+          command: yarn install --frozen-lockfile
+      - run:
+          name: lint code
+          command: yarn prettier --write --list-different '**/*.+(json|js)'
+
+  lint-lambda-medialive:
+    docker:
+      - image: circleci/node:12.20
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+      # Run on the highest version of node available on AWS Lambda
+      # https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime
+    working_directory: ~/marsha/src/aws/lambda-medialive
+    steps:
+      - checkout:
+          path: ~/marsha
+      - restore_cache:
+          keys:
+            - v3-lambdas-medialive-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install complete lambda dependencies
+          command: yarn install --frozen-lockfile
+      - run:
+          name: lint code
+          command: yarn prettier --write --list-different '**/*.+(json|js)'
+
+  lint-lambda-medialive-routing:
+    docker:
+      - image: circleci/node:12.20
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+      # Run on the highest version of node available on AWS Lambda
+      # https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime
+    working_directory: ~/marsha/src/aws/lambda-medialive-routing
+    steps:
+      - checkout:
+          path: ~/marsha
+      - restore_cache:
+          keys:
+            - v3-lambdas-medialive-routing-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install complete lambda dependencies
+          command: yarn install --frozen-lockfile
+      - run:
+          name: lint code
+          command: yarn prettier --write --list-different '**/*.+(json|js)'
```

Comment:
I think there might be a discrepancy between local and CI runs since you don't seem to use versioned prettier configuration in the CI.

Reply:
I don't understand your comment. The configuration is at `src/.pretierrc`, prettier is looking for the configuration like this : 
> The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found.

So we will use the same configuration file locally and in the CI

Reply:
Wow, didn't know about this. This is an uncommon behavior. It means it can explore upper from `$PWD` :exploding_head: